<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>placa_mae_window</class>
 <widget class="QDialog" name="placa_mae_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1458</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>-10</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>placas mae</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_placa_mae">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_placa_mae_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_placa_mae_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_placa_mae_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>450</y>
     <width>391</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelplacamae">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelplacamae_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>400</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelplacamae_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>800</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelplacamae_4">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>800</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_carrinho">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>0</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>carrinho R$ </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
